<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>setQuiz</class>
 <widget class="QDialog" name="setQuiz">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>267</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
font:20px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>315</width>
        <height>1692</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox_16">
         <property name="title">
          <string>Question 1</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_36">
          <item>
           <widget class="QPlainTextEdit" name="plainTextEdit101"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_16">
            <item>
             <widget class="QCheckBox" name="checkBox_111">
              <property name="text">
               <string>a</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_112">
              <property name="text">
               <string>b</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_113">
              <property name="text">
               <string>c</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_114">
              <property name="text">
               <string>d</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_101">
              <property name="text">
               <string>Add</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Question2</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_22">
          <item>
           <widget class="QPlainTextEdit" name="plainTextEdit102"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QCheckBox" name="checkBox_121">
              <property name="text">
               <string>a</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_122">
              <property name="text">
               <string>b</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_123">
              <property name="text">
               <string>c</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_124">
              <property name="text">
               <string>d</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton102">
              <property name="text">
               <string>Add</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_14">
         <property name="title">
          <string>Question3</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_34">
          <item>
           <widget class="QPlainTextEdit" name="plainTextEdit_103"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_14">
            <item>
             <widget class="QCheckBox" name="checkBox_131">
              <property name="text">
               <string>a</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_132">
              <property name="text">
               <string>b</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_133">
              <property name="text">
               <string>c</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_134">
              <property name="text">
               <string>d</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton103">
              <property name="text">
               <string>Add</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_12">
         <property name="title">
          <string>Question4</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_32">
          <item>
           <widget class="QPlainTextEdit" name="plainTextEdit_104"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <widget class="QCheckBox" name="checkBox_141">
              <property name="text">
               <string>a</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_142">
              <property name="text">
               <string>b</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_143">
              <property name="text">
               <string>c</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_144">
              <property name="text">
               <string>d</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton104">
              <property name="text">
               <string>Add</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Question5</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_23">
          <item>
           <widget class="QPlainTextEdit" name="plainTextEdit_105"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QCheckBox" name="checkBox_151">
              <property name="text">
               <string>a</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_152">
              <property name="text">
               <string>b</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_153">
              <property name="text">
               <string>c</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_154">
              <property name="text">
               <string>d</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton105">
              <property name="text">
               <string>Add</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_18">
         <property name="title">
          <string>Question6</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_38">
          <item>
           <widget class="QPlainTextEdit" name="plainTextEdit_106"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_18">
            <item>
             <widget class="QCheckBox" name="checkBox_161">
              <property name="text">
               <string>a</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_162">
              <property name="text">
               <string>b</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_163">
              <property name="text">
               <string>c</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_164">
              <property name="text">
               <string>d</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton106">
              <property name="text">
               <string>Add</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_11">
         <property name="title">
          <string>Question7</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_31">
          <item>
           <widget class="QPlainTextEdit" name="plainTextEdit_107"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <widget class="QCheckBox" name="checkBox_171">
              <property name="text">
               <string>a</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_172">
              <property name="text">
               <string>b</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_173">
              <property name="text">
               <string>c</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_174">
              <property name="text">
               <string>d</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton107">
              <property name="text">
               <string>Add</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_13">
         <property name="title">
          <string>Question8</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_33">
          <item>
           <widget class="QPlainTextEdit" name="plainTextEdit_108"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_13">
            <item>
             <widget class="QCheckBox" name="checkBox_181">
              <property name="text">
               <string>a</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_182">
              <property name="text">
               <string>b</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_183">
              <property name="text">
               <string>c</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_184">
              <property name="text">
               <string>d</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton108">
              <property name="text">
               <string>Add</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_15">
         <property name="title">
          <string>Question9</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_35">
          <item>
           <widget class="QPlainTextEdit" name="plainTextEdit_109"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_15">
            <item>
             <widget class="QCheckBox" name="checkBox_191">
              <property name="text">
               <string>a</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_192">
              <property name="text">
               <string>b</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_193">
              <property name="text">
               <string>c</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_194">
              <property name="text">
               <string>d</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton109">
              <property name="text">
               <string>Add</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Question10</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_24">
          <item>
           <widget class="QPlainTextEdit" name="plainTextEdit_110"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QCheckBox" name="checkBox_101">
              <property name="text">
               <string>a</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_102">
              <property name="text">
               <string>b</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_103">
              <property name="text">
               <string>c</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_104">
              <property name="text">
               <string>d</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton110">
              <property name="text">
               <string>Add</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Submit Quiz</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
